### ISSUE TITLE
When recovering password: "record not found" message is insecure

### ISSUE THREAD

**Issue by berkus** [pepyatka/pepyatka-html#581]
Lets people probe for registered emails.

**Comment by epicmonkey** [comment]
Not quite, you could scan for emails already over sign up/settings pages, i.e. you should not be able to change your email to one that's already in the system.
